# Exercício 01 — Classificação de carga em cabos de aço

**Aluno:** AUGUSTO CAMPOS RINCO  
**Curso:** Engenharia Civil

## Descrição do problema

Este programa irá analisar a condição de operação de um cabo de aço submetido a uma determinada carga. Para isso, serão calculadas a área da seção transversal, a tensão aplicada e o fator de utilização do cabo. Ao final, o programa classificará o nível de risco da operação.

Código: dados de entrada

Os valores fornecidos são diâmetro de 12 mm, carga de 8.500 N e tensão de ruptura de 400 MPa.

In [4]:
# Dados de entrada

diametro_mm = 12           # Diâmetro do cabo em milímetros (mm)
carga_N = 8500             # Força aplicada no cabo em Newtons (N)
tensao_ruptura_MPa = 400   # Tensão de ruptura do aço em Megapascals (MPa)

In [5]:
import math

# Conversão do diâmetro de milímetros para metros
diametro_m = diametro_mm / 1000

# Cálculo da área da seção transversal circular
area_m2 = math.pi * (diametro_m / 2) ** 2

# Cálculo da tensão aplicada em Pascal
tensao_Pa = carga_N / area_m2

# Conversão da tensão de Pa para MPa
tensao_MPa = tensao_Pa / 1_000_000

# Resultados intermediários
print(f"Diâmetro convertido: {diametro_m:.4f} m")
print(f"Área da seção transversal: {area_m2:.6f} m²")
print(f"Tensão aplicada: {tensao_MPa:.2f} MPa")

Diâmetro convertido: 0.0120 m
Área da seção transversal: 0.000113 m²
Tensão aplicada: 75.16 MPa


In [6]:
# Cálculo do fator de utilização
fator_utilizacao = tensao_MPa / tensao_ruptura_MPa

# Classificação do risco
if fator_utilizacao < 0.50:
    classificacao = "Operação segura"
elif fator_utilizacao < 0.75:
    classificacao = "Atenção — verificar periodicamente"
elif fator_utilizacao < 0.90:
    classificacao = "Risco elevado — substituição recomendada"
else:
    classificacao = "Perigo — cabo fora de operação"

print(f"Fator de utilização: {fator_utilizacao:.2f}")
print(f"Classificação: {classificacao}")

Fator de utilização: 0.19
Classificação: Operação segura


In [7]:
print("=== RELATÓRIO DE ANÁLISE DE CABO ===")
print(f"Diâmetro:            {diametro_mm:.2f} mm")
print(f"Carga aplicada:      {carga_N:.2f} N")
print(f"Área da seção:       {area_m2:.6f} m²")
print(f"Tensão aplicada:     {tensao_MPa:.2f} MPa")
print(f"Fator de utilização: {fator_utilizacao:.2f}")
print(f"Classificação:       {classificacao}")
print("=====================================")

=== RELATÓRIO DE ANÁLISE DE CABO ===
Diâmetro:            12.00 mm
Carga aplicada:      8500.00 N
Área da seção:       0.000113 m²
Tensão aplicada:     75.16 MPa
Fator de utilização: 0.19
Classificação:       Operação segura


## Interpretação do resultado

O cabo encontra-se em condição segura de operação, pois o fator de utilização calculado foi de aproximadamente 0,19, ficando abaixo do limite de 0,50 estabelecido para operação segura.

Isso significa que a tensão aplicada corresponde a aproximadamente 19% da tensão de ruptura do material. Recomenda-se ao engenheiro responsável manter as inspeções e manutenções periódicas do cabo, verificando possíveis sinais de desgaste, corrosão ou danos que possam comprometer sua utilização.